src/StaticAnalysisResultsInterchangeFormat/IResultLogWriter.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.CodeAnalysis.StaticAnalysisResultsInterchangeFormat.DataContracts;
+
+namespace Microsoft.CodeAnalysis.StaticAnalysisResultsInterchangeFormat
+{
+    /// <summary>This interface serves as a sink for <see cref="IssueLog"/> format issues.</summary>
```

Comment:
`IssueLog` => `ResultLLog`, except I think the whole sentence should be "Interface that serves as a sink for SARIF results"